dev/reference/tools/tidb-lightning/config.md

```diff
@@ -0,0 +1,324 @@
+---
+title: TiDB Lightning Configuration
+summary: CLI usage and sample configuration
+category: reference
+---
+
+# TiDB Lightning Configuration
+
+## Configuration files
+
+TiDB Lightning's global and task configurations have compatible structures.
+Unless [Server mode](/reference/tools/tidb-lightning/web.md) is enabled, the same file is used for both parts of configuration.
+
+### TiDB Lightning (Global)
+
+```toml
+### tidb-lightning global configuration
+
+[lightning]
+# the HTTP port for web interface and Prometheus metrics pulling (0 to disable)
+status-addr = ':8289'
+
+# Toggle server mode and use of the web interface.
+# See the corresponding section for details.
+server-mode = false
+
+# Logging
+level = "info"
+file = "tidb-lightning.log"
+max-size = 128 # MB
+max-days = 28
+max-backups = 14
+```
+
+### TiDB Lightning (Task)
+
+```toml
+### tidb-lightning task configuration
+
+[lightning]
+# Checks if the cluster satisfies the minimum requirement before starting.
+#check-requirements = true
+
+# The maximum number of engines to be opened concurrently.
+# Each table is split into one "index engine" to store indices, and multiple
+# "data engines" to store row data. These settings control the maximum
+# concurrent number for each type of engines.
+# These values affect the memory and disk usage of tikv-importer.
+# The sum of these two values must not exceed the max-open-engines setting
+# for tikv-importer.
+index-concurrency = 2
+table-concurrency = 6
+
+# The concurrency number of data. It is set to the number of logical CPU
+# cores by default. When deploying together with other components, you can
+# set it to 75% of the size of logical CPU cores to limit the CPU usage.
+#region-concurrency =
```

Comment:
I'm not sure about this. Is it a typo?
```suggestion
region-concurrency =
```

Reply:
No, it is calculated dynamically, so a concrete number cannot be placed here.

Reply:
@kennytm Got it. And do we need the `#` character at the beginning (like a code comment)?

Reply:
@lilin90 Well this is exactly a comment.

Removing the `#` would cause the file to become invalid.

Reply:
@kennytm Got it. ^_^ Thanks!